src/components/SignersPanel/index.tsx

```diff
@@ -0,0 +1,93 @@
+import { useProject } from 'providers/Project/projectHooks';
+import React, { useState, useMemo } from 'react';
+import theme from '../../theme';
+import { CollapseButton } from './CollapseButton';
+import AccountPicker from 'components/AccountPicker';
+import { Flex, Text } from 'theme-ui';
+import {
+  SignersContainer,
+} from '../Arguments/styles';
+import { Account } from 'api/apollo/generated/graphql';
+import Avatar from 'components/Avatar';
+
+type SignersProps = {
+  maxSelection?: number;
+  selected: number[];
+  updateSelectedAccounts: (selection: number[]) => void;
+};
+
+const displayAddress = ({ address }: { address: string }) => {
+  return `0x${address.slice(-2)}`;
+}
+
+const AvatarIcon = (seed: number, index: number, complete: boolean) => {
+  return (
+    <Avatar
+      seed={seed}
+      index={index}
+      style={{
+        width: '17px',
+        height: '17px',
+        display: 'block',
+        borderRadius: '20px',
+        border: `1px solid ${complete ? theme.colors.primary : theme.colors.errors}`
+      }}
+    />)
+}
+
+const AvatarIconList = (seed: number, indexes: number[], complete: boolean = false) => {
+  return (
+    <Flex>
+      {indexes.map(index => {
+        return AvatarIcon(seed, index, complete);
+      })}
+    </Flex>
+  )
+}
+
+const PanelHeader = (maxSelection: number, accounts: Account[], seed: number, selected: number[] = []) => {
+  // TODO: accounts might be needed for displaying account addr, to be determined
+  let message = ""
+  const correctNumSigners = selected.length === maxSelection;
+  const signersSelected = "Signers Selected";
```

Comment:
is it best practice to save text constant like this? and centralize constants in their own file?

 `const SIGNERS_SELECTED = "Signers Selected";`